packages/mural-picker/src/components/mural-picker/index.tsx

```diff
@@ -262,12 +262,20 @@ const MuralPicker = ({
     }
   }, [rooms]);
 
-  const templateSearch = (q?: string) => {
-    if (q) {
-      const qEncoded = encodeURIComponent(q);
-      return apiClientRef.current.searchCrossWorkspaceTemplates({
-        q: qEncoded,
-      });
+  const templateSearch = async (searchKeyword?: string) => {
+    let templateResults: Template[] | TemplateSummary[];
```

Comment:
**Nitpick**: this variable can be removed

Reply:
good idea!